modules/virt-configuring-vm-with-node-exporter-service.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events-monitoring/virt-exposing-custom-metrics-for-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-vm-with-node-exporter-service_{context}"]
+= Configuring a virtual machine with the node exporter service
+
+Download the `node-exporter` file on to the virtual machine. Then, create a `systemd` service that runs the node-exporter service when the virtual machine boots.
+
+.Prerequisites
+* The pods for the component are running in the `openshift-user-workload-monitoring` project.
+* Grant the `monitoring-edit` role to users who need to monitor this user-defined project.
+
+.Procedure
+
+. Log on to the virtual machine.
+
+. Download the `node-exporter` file on to the virtual machine by using the directory path that applies to the version of `node-exporter` file.
++
+[source,terminal]
+----
+$ wget https://github.com/prometheus/node_exporter/releases/download/v1.3.1/node_exporter-1.3.1.linux-amd64.tar.gz
+----
+
+. Extract the executable and place it in the `/usr/bin` directory.
++
+[source,terminal]
+----
+$ sudo tar xvf node_exporter-1.3.1.linux-amd64.tar.gz \
+    --directory /usr/bin --strip 1 "*/node_exporter"
+----
+
+. Create a `node_exporter.service` file in this directory path: `/etc/systemd/system`. This `systemd` service file runs the node-exporter service when the virtual machine reboots.
```

Comment:
Somehow the rest of the module text is displayed bold although it renders okay

Reply:
This particular text is in regular font, not bold so it is odd that GitHub displays it in bold although the HTML renders normal.